django_cloud_deploy/skeleton/source_generator.py

```diff
@@ -694,13 +694,15 @@ def install_requirements(self, project_dir: str):
         """
         requirements_path = os.path.join(project_dir, 'requirements.txt')
         try:
-            subprocess.check_call(
+            subprocess.call(
                 ['python3', '-m', 'pip', 'install', '-r', requirements_path],
                 stdout=subprocess.DEVNULL,
                 stderr=subprocess.DEVNULL)
         except (SystemExit, subprocess.CalledProcessError) as e:
-            raise crash_handling.UserError(
-                'Not able to install Django project dependencies.') from e
+            print(('Fail to install some packages listed in {}. This may or '
```

Comment:
This change is made because our tool will install all the packages, but some of them may not be required for deployment. For example, "mysqlclient". When installing "mysqlclient" on CircleCI, we will see ```EnvironmentError: mysql_config not found```. But actually "mysqlclient" is not required for deployment.

Reply:
:s/fail/failed

Reply:
the errors. -> any errors

Reply:
Done